scanpy/neighbors/__init__.py

```diff
@@ -705,9 +629,14 @@ def compute_neighbors(
         # neighbor search
         use_dense_distances = (metric == 'euclidean' and X.shape[0] < 8192) or knn == False
         if use_dense_distances:
-            # standard eulcidean case for relatively small matrices
-            self._distances, knn_indices, knn_distances = compute_neighbors_numpy(
-                X, n_neighbors, knn=knn)
+            _distances = pairwise_distances(X, metric=metric, **metric_kwds)
```

Comment:
@VolkerBergen it's here where the changed implementation sometimes raises an sklearn warning. (FYI @ivirshup)

I'm not sure why this happens. One could suppress the warning in this single line, but of course that's a hack.

Reply:
Would you mind sharing the warning you get?

Reply:
Apologies for the late response. The warning is the following:

```
./site-packages/sklearn/metrics/pairwise.py:257: RuntimeWarning: invalid value encountered in sqrt
  return distances if squared else np.sqrt(distances, out=distances)
```

It is also an issue for UMAP (https://github.com/lmcinnes/umap/issues/87) which prob traces back to some neighbors computing as well.

Reply:
That might be it... it should be a temporary thing... It's a bit ugly to have this warning every now and then but it's fine if it's temporary I'd say; if it doesn't go away we'll make a PR to sckit-learn at some point.